spec/sample-data/valid/trademark-02.json

```diff
@@ -0,0 +1,82 @@
+{
+  "holders": [{
+    "entity_type": "unknown",
+    "entity_properties": {
+      "name": "L'ORÉAL, Société anonyme",
+      "addresses": [{
+        "address": {
+          "street_address": "14, rue Royale, F-75008 PARIS",
+          "country_code": "FR"
+        },
+        "type": "unknown"
+      }],
+      "jurisdiction": "FRANCE"
+    }
+  }],
+  "register": "USPTO",
```

Comment:
@thomasmarshall I would use uspto_trademarks for this one, and wipo_madrid for the WIPO example (both USPTO have other registers apart from the trademarks ones).